web/crux-ui/e2e/deployment-copibility.spec.ts

```diff
@@ -153,7 +153,7 @@ test.describe('Complex product', () => {
 
     await expect(await page.locator('button:has-text("Copy")')).toHaveCount(1)
 
-    const navigation = page.waitForNavigation({ url: `**${versionUrl(productId, versionId)}/deployments/**` })
+    const navigation = page.waitForURL(`**${versionUrl(productId, versionId)}/deployments/**`)
```

Comment:
According to the docs, `waitForUrl()` don't need to be initialized before the action which triggers the navigation.